models/daily_spend.sql

```diff
@@ -113,7 +116,8 @@ adj_for_incl_cloud_services_daily as (
         null as storage_type,
         null as warehouse_name,
         null as database_name,
-        coalesce(sum(stg_metering_daily_history.credits_adjustment_cloud_services * daily_rates.effective_rate), 0) as spend
+        coalesce(sum(stg_metering_daily_history.credits_adjustment_cloud_services * daily_rates.effective_rate), 0) as spend,
+        0 as spend_net_cloud_services
```

Comment:
I've set this to 0 because if somewhere were to do:

```sql
select
     date,
     sum(spend_net_cloud_services) as spend
from daily_spend
group by 1,2
```

instead of

```sql
select
     date,
     sum(spend) as spend
from daily_spend
group by 1,2
```

they **should** get the same answer. will validate.

Reply:
they line up perfectly on our account.